.github/workflows/pulp_ci.yaml

```diff
@@ -35,6 +41,29 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install tox
+      - name: Travis to GH actions
+        run: |
+          find ./ -type f -exec sed -i -e 's/TRAVIS_BUILD_DIR/GITHUB_WORKSPACE/g' {} \;
+          find ./ -type f -exec sed -i -e 's/TRAVIS_EVENT_TYPE/GITHUB_EVENT_NAME/g' {} \;
+          find ./ -type f -exec sed -i -e 's/TRAVIS_BRANCH/\{GITHUB_REF##*\/\}/g' {} \;
+          find ./ -type f -exec sed -i -e 's/TRAVIS_PULL_REQUEST_BRANCH/\{GITHUB_REF##*\/\}/g' {} \;
+          find ./ -type f -exec sed -i -e 's/TRAVIS_TAG/\{GITHUB_REF##*\/\}/g' {} \;
+          find ./ -type f -exec sed -i -e 's/-z "$TRAVIS_TAG" //g' {} \;
+          find ./ -type f -exec sed -i -e 's/${TRAVIS_PYTHON_VERSION}/3.7/g' {} \;
+          find ./ -type f -exec sed -i -e 's/"$TRAVIS_PULL_REQUEST" != "false"/"$GITHUB_EVENT_NAME" = "pull_request"/g' {} \;
+          find ./ -type f -exec sed -i -e 's/"$TRAVIS_PULL_REQUEST" = "false"/"$GITHUB_EVENT_NAME" != "pull_request"/g' {} \;
+          find ./ -type f -exec sed -i -e 's/travis_fold:start:/::group::/g' {} \;
+          find ./ -type f -exec sed -i -e 's/travis_fold:end:$containerlog/::endgroup::/g' {} \;
+          find ./ -type f -exec sed -i -e 's/.\/.travis\/check_commit.sh/#.\/.travis\/check_commit.sh/g' {} \;
+      - name: Before Install
+        run: |
+          .travis/before_install.sh
+          echo "::set-env name=TEST::${{ matrix.env.TEST }}"
+          cd ..
+          find ./ -type f -exec sed -i -e 's/timeout 90 bash/timeout 120 bash/g' {} \;
+          find ./ -type f -exec sed -i -e 's/\/usr\/local\/bin\/kubectl/$(which kubectl)/g' {} \;
+          find ./ -type f -exec sed -i -e 's/# containers are not running yet./sudo cp -i \/etc\/rancher\/k3s\/k3s.yaml $HOME\/.kube\/config/g' {} \;
+          cd pulp_rpm_prerequisites
```

Comment:
I believe we don't need it here, as the `Required PR` thing is just for PRs

Reply:
I modified the script (and moved it from .travis/ to .github/)